webcompat/static/css/webcompat.dev.css

```diff
@@ -40,18 +40,18 @@
 
 #body-webcompat {
   background-color: var(--color-bg-page);
-  margin-top: var(--navi-height);
+  padding: var(--navi-height) var(--grid-edge-s) 25px;
```

Comment:
could we change the `25px` to `--unit-gutter-m` as this should work context wise?